src/swap/components/SwapSettingsSlippageDescription.tsx

```diff
@@ -0,0 +1,18 @@
+import { cn } from '../../styles/theme';
+import type { SwapSettingsSlippageDescriptionReact } from '../types';
+
+export function SwapSettingsSlippageDescription({
+  children,
+  className,
+}: SwapSettingsSlippageDescriptionReact) {
+  return (
+    <p
+      className={cn(
+        'mb-2 font-normal font-sans text-gray-600 text-xs leading-4 dark:text-gray-400"',
```

Comment:
`dark` color should be automatic, why is here custom?

Create this fork in the theme, make harder for developer to do custom themes at global level.

Reply:
That makes sense, I refactored and removed the classes `text-gray-600` and `dark:text-gray-400` and I added `text-ock-foreground-muted`.

I've confirmed that this is working in the playground.